taccsite_cms/static/site_cms/css/src/_imports/objects/o-section.css

```diff
@@ -242,19 +221,15 @@ Styleguide Objects.Section
 /* FP-1462: Extract to new banner component(s) */
 
 /* HACK: To hide vertical overflow by placing sibling sections atop */
+[class*="o-section--style"] { position: relative; }
```

Comment:
**Explanation.** This hack was relying on `position: relative` from code above (deleted). _Now `position: relative` can exist here, cuz this is the only code that needs it._